go/vt/sqlparser/analyzer.go

```diff
@@ -205,6 +210,8 @@ func (s StatementType) String() string {
 		return "PRIV"
 	case StmtExplain:
 		return "EXPLAIN"
+	case StmtSavepoint:
+		return "SAVEPOINT"
```

Comment:
#6414 This PR has this change. Once it is merged you would need to rebase it.

Reply:
got it, thx